packages/tools/src/types/SegmentationStateTypes.ts

```diff
@@ -77,6 +77,10 @@ export type Segmentation = {
    * only labelmap representation is supported)
    */
   representationData: SegmentationRepresentationData;
+  /**
```

Comment:
Should this just be FOR?  Is there any need to have the full volume id?

Reply:
Put it inside the function but I added the volumeOfRefrenceId so I dont want the applications to break if they did not set it